src/BizHawk.Emulation.Cores/Consoles/Nintendo/Gameboy/Gambatte.ISettable.cs

```diff
@@ -255,14 +255,21 @@ public int LatchedRTCSeconds
 				get => _latchedRTCSeconds;
 				set => _latchedRTCSeconds = Math.Max(0, Math.Min(63, value));
 			}
+			
+			public enum FrameLengthType
+			{
+				VBlankDrivenFrames,
+				EqualLengthFrames,
+				UserDefinedFrames
+			}
 
-			[DisplayName("Equal Length Frames")]
-			[Description("When false, emulation frames sync to vblank.  Only useful for high level TASing.")]
-			[DefaultValue(false)]
-			public bool EqualLengthFrames
+			[DisplayName("Frame Length")]
+			[Description("Sets how long an emulation frame will last.\nVBlank Driven Frames will make emulation frames sync to VBlank. Recommended for TASing.\nEqual Length Frames will force all frames to emit 35112 samples. Legacy, not recommended for TASing.\nUser Defined Frames allows for the user to define how many samples are emitted for each frame. Only useful if sub-frame input is desired.")]
+			[DefaultValue(FrameLengthType.VBlankDrivenFrames)]
+			public FrameLengthType FrameLength
 			{
-				get => _equalLengthFrames;
-				set => _equalLengthFrames = value;
+				get => _frameLength;
+				set => _frameLength = value;

```

Comment:
I think this can be an auto-prop (`public FrameLengthType FrameLength { get; set; }`). Elsewhere in this file, a backing field is used because the property's setter has extra validation, but not here.